Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -2864,171 +3212,453 @@ def extract_sentinelone_zip_file(zip_file_data: bytes, password: str) -> Tuple[s
+    demisto.debug("with in the set xsoar incident entries method")
+    if mirrored_object.get("threatInfo", {}).get("incidentStatus") == "resolved":
+        demisto.debug(f"Incident is closed: {remote_incident_id}")
+        entries.append(
+            {
+                "Type": EntryType.NOTE,
+                "Contents": {
+                    "dbotIncidentClose": True,
+                    "closeReason": "Incident was closed on SentinelOne",
+                },
+                "ContentsFormat": EntryFormat.JSON,
+            }
+        )
+        return entries
+    elif mirrored_object.get("threatInfo", {}).get("incidentStatus") in (
+        set(INCIDENT_STATUS) - {"resolved"}
+    ):
+        demisto.debug(f"Incident is reopened: {remote_incident_id}")
+        entries.append(
+            {
+                "Type": EntryType.NOTE,
+                "Contents": {"dbotIncidentReopen": True},
+                "ContentsFormat": EntryFormat.JSON,
+            }
+        )
+        return entries
+
+
+def get_remote_incident_data(client: Client, remote_incident_id: str):
+    """
+    Called every time get-remote-data command runs on an incident.
+    Gets the relevant incident entity from the remote system (SentinelOne). The remote system returns a list with this
+    entity in it. We take from this entity only the relevant incoming mirroring fields, in order to do the mirroring.
+    """
+    mirrored_data_list = client.get_s1_threats_information(
+        remote_incident_id
+    )  # a list with one dict in it
+    mirrored_data = mirrored_data_list[0]
+
+    mirrored_data["incident_type"] = "incident"
+    return mirrored_data
+
+
+def get_remote_data_command(client: Client, args: dict):
+    """
+    get-remote-data command: Returns an updated remote incident.
+    Args:
+        args:
+            id: incident id to retrieve.
+            lastUpdate: when was the last time we retrieved data.
+
+    Returns:
+        GetRemoteDataResponse object, which contain the incident data to update.
+    """
+    demisto.debug("calling get remote data command")
+    remote_args = GetRemoteDataArgs(args)
+    remote_incident_id = remote_args.remote_incident_id
 
+    mirrored_data = {}
+    entries: List = []
+    try:
+        demisto.debug(
+            f"Performing get-remote-data command with incident id: {remote_incident_id} "
+            f"and last_update: {remote_args.last_update}"
+        )
+        mirrored_data = get_remote_incident_data(client, remote_incident_id)
+        demisto.debug(f"mirrored object in get-remote-data {mirrored_data}")
+        if mirrored_data:
+            demisto.debug(f"Update incident {remote_incident_id} with fields: {mirrored_data}")
+            entries = set_xsoar_incident_entries(mirrored_data, entries, remote_incident_id)  # sets in place
+        if not mirrored_data:
+            demisto.debug(f"No delta was found for detection {remote_incident_id}.")
```

Comment:
```suggestion
            demisto.debug(f"No delta was found for incident {remote_incident_id}.")
```
Notice that all the 3 logs here are not needed and are repetitive.